.circleci/config.yml

```diff
@@ -59,7 +59,7 @@ workflows:
   lint_and_build:
     jobs:
       - lint
```

Comment:
can you remove this until it is passing?

Reply:
Sure can!